stats/nba_sql.py

```diff
@@ -100,23 +105,24 @@ def main():
         play_by_play_requester,
         pgtt_requester
     ]
-    do_create_schema(create_schema, object_list)
 
-    populate_base_tables(
-        do_base_tables,
-        seasons,
-        request_gap,
-        team_requester,
-        player_requester,
-        event_message_type_builder)
+    if create_schema:
+        do_create_schema(object_list)
+
+    if not skip_base_tables:
+        populate_base_tables(
+            seasons,
+            request_gap,
+            team_requester,
+            player_requester,
+            event_message_type_builder)
```

Comment:
Much cleaner, I'm in the Erlang headspace where booleans can be checked in the function head but I'm sure this is more idiomatic for python

Reply:
I see, yeah that does explain a bit of the conventions used. I don't mean that in a bad way, just that it's very glaring, and I am resisting the urge to _fix_ all the gaps. 😃 

Python is my domain, so I can help with a lot of this stuff. Flake8 is a widely used python linter and syntax checker. You can run it once to see what I mean. I can also help with packaging, which this project has a very good use case for.

From my side, I really appreciate this project since I wanted to sharpen my SQL, and DB modelling skills. I really started using it yesterday when I got into an argument with some one on Reddit 😆

I am beginning to see how this data is presented, and how I need to extract stuff to see patterns, and what more I need. I am not aware of any of the tools you talked about in the wiki, so I will eventually give those a try too.

So it's a very cool project.

Reply:
Yeah, this is my first python project so I'm sure it is nasty to look at. I'll give Flake8 a spin on it and see what I can fix.

Glad it is helpful to you so far, it was a pandemic project for me to stave off boredom! If you really want to get deep, checkout the Superset 'tutorial' I added. You can get some great data vis from this DB. And once I've gotten around to adding box score, some _really_ wild analytics available.

Let me know if you have any thoughts around the broader initiatives that I added as issues. This was meant to be more of a analytic database that could be hooked up to Redshift / Greenplum, but I haven't gotten quite that far yet.

Reply:
Damn, flake8 reports 602 violations. Alot seem to be from star importing, because I was lazy. I'll chip away at a few of these.